src/cli/tc.rs

```diff
@@ -20,10 +20,57 @@ use crate::core::config::reader::ConfigReader;
 use crate::core::http::API_URL_PREFIX;
 use crate::core::print_schema;
 use crate::core::rest::{EndpointSet, Unchecked};
+use crate::core::runtime::TargetRuntime;
+
 const FILE_NAME: &str = ".tailcallrc.graphql";
 const YML_FILE_NAME: &str = ".graphqlrc.yml";
 const JSON_FILE_NAME: &str = ".tailcallrc.schema.json";
 
+const GRAPHQL: &str = "GraphQL";
+const JSON: &str = "JSON";
+const YML: &str = "YML";
+
+const MAIN_GRAPHQL: &str = r#"
+schema {
+  query: Query
+}
+type Query {
+  greet: String @expr(body: "Hello World!")
+}
+"#;
+
+const MAIN_JSON: &str = r#"
+{
+  "schema": {
+    "query": "Query"
+  },
+  "types": {
+    "Query": {
+      "fields": {
+        "greet": {
+          "type": "String",
+          "expr": {
+            "body": "Hello World!"
+          }
+        }
+      }
+    }
+  }
+}
+"#;
+
+const MAIN_YML: &str = r#"
+schema:
+  query: Query
+types:
+  Query:
+    fields:
+      greet:
+        type: String
+        expr:
+          body: Hello World!
+"#;
```

Comment:
Use the rust API for Config to build a default config once and encode it to different formats using to_json and to_sdl methods.

Reply:
done